devs.html

```diff
@@ -59,6 +59,12 @@ <h4>Neill Lambert</h4>
         <h5>RIKEN</h5>
         <h6>Main focus: Non-Markovian dynamics, counting statistics.</h6>
     </div>
+    <div class="col-sm-3">
+        <img src="images/boxi.jpg" class="img-polaroid">
+        <h4>Boxi Li</h4>
+        <h5>ETH Zurich</h5>
+        <h6>Main focus: Quantum repeater, quantum optimal control</h6>
```

Comment:
I think that the expertise should be really related more with regard to Qutip development than for your research. 

Basically it is also a way to already direct requests from users. In your case, I think you could highlight the pulse-level control of quantum circuits. I'm not sure I'd include quantum repeaters here. 

Remember to update your affiliation in the near future.

Reply:
I see, updated it with "quantum device simulation". Thanks